go/src/koding/config/config.json

```diff
@@ -55,6 +55,50 @@
         "url": "https://dev-p2.koding.com/-/ipcheck"
       }
     ],
+    "kd-latest": [
```

Comment:
Pawel is already working on this (https://github.com/koding/koding/issues/8920).